reports/California.md

```diff
@@ -380,6 +380,44 @@ id: ca-losangeles-26
+
+id: ca-losangeles-19
+
+**Links**
+* [Complaint in NLG lawsuit](https://nlg-la.org/wp-content/uploads/sites/5/2020/06/AMENDED-COMPLAINT-ECF.pdf)
+* [Allegations by protester Laura Montilla](https://www.instagram.com/p/CBBNXXkJs0a/?igshid=jgeposybda4a)
+* [Relevant video](https://www.instagram.com/p/CBEr2emJRyJ/?igshid=oloj7lgtv81t)
+* [Local news coverage mentioning the incident](https://abc7.com/protest-arrest-george-floyd-curfews/6234180/)
+* [LA Times article mentioning both lawsuits](https://www.latimes.com/california/story/2020-06-22/complaints-lapd-abuses-mount-protest-lawsuit)
+* [UCLA disapproves of use of Jackie Robinson stadium for a field jail](https://twitter.com/UCLA/status/1268039753015812096)
+* [First response by UCLA professors, containing further allegations of misconduct by LAPD and LASD](https://twitter.com/ananyaUCLA/status/1267985147225231360)
+* [Second response by UCLA professors](https://drive.google.com/file/d/157MdwMMCEebcUyesO9HMOYK9p21q3gpE/view)
+* https://www.reddit.com/r/LosAngeles/comments/gxi8b2/how_is_noone_talking_about_this_women_from/
+* [Allegation of LAPD cutting someone's hand while trying to remove a zip tie](https://www.reddit.com/r/LosAngeles/comments/gy3e3r/the_lapd_has_been_slicing_into_protesters_hands/ft9so6j/?context=8&depth=9)
+* https://abc7.com/uclas-jackie-robinson-stadium-used-as-field-jail-during-protests-/6229078/
+* https://dailytrojan.com/2020/06/09/police-protocol-compromises-safety-of-student-protesters/
+
+
+### Police arrest protesters en masse and detain them in poor conditions | June 2nd
+
+Police arrested protesters en masse near the home of LA mayor Eric Garcetti and in other parts of the city, zip tied their hands, and held them for extended periods without appropriate social distancing. At least one arrestee alleges that protesters were given explicit permission to remain past the curfew, then arrested for being out after curfew. Three arrestees describe in similar terms arrestees (at least one of whom was gender nonconforming) being sexually harrassed by officers. Many arrestees say they were released in the middle of the night, without transportation home, in unfamiliar parts of the city. Two arrestees describe in similar terms police mocking a group of detainees and refusing to release them until they were "good girls" or "reasonable." One detainee says this was due to some detainees removing zip ties using lighters because the ties were dangerously tight. Another detainee says that it was due to detainees requesting medical attention and other improvements in their conditions, which would constitute first-amendment-protected speech. One arrestee allegedly should have been exempt from the curfew as they were returning home from work. A lawsuit has been filed in connection with these and other abuses by the National Lawyers' Guild. (See also incidents ca-losangeles-19 (near-identical behavior the previous day) and ca-santamonica-2 (similar behavior in Santa Monica).)
```

Comment:
```Three arrestees describe in similar terms arrestees (at least one of whom was gender nonconforming) being sexually harrassed by officers.```

This sentence is a mouthful. Could you try re-working it for a little more clarity?